### PR TITLE
[Doc] Add notes to Async MV

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md [hunk 1]
```diff
@@ -168,6 +168,10 @@ Comment on the materialized view. Note that `COMMENT` must be placed after `mv_n
 
 The bucketing strategy of the asynchronous materialized view. StarRocks supports hash bucketing and random bucketing (from v3.1 onwards). If you do not specify this parameter, StarRocks uses the random bucketing strategy and automatically sets the number of buckets.
 
+> **NOTE**
+>
+> While creating an asynchronous materialized view, you must specify either the `distribution_desc` or the `refresh_scheme`, or both.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md:173]: you must specify either `distribution_desc` or `refresh_scheme`

File: docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md [hunk 2]
```diff
@@ -203,7 +207,11 @@ The refresh moment of the materialized view. Default value: `IMMEDIATE`. Valid v
 - `IMMEDIATE`: Refresh the asynchronous materialized view immediately after it is created.
 - `DEFERRED`: The asynchronous materialized view is not refreshed after it is created. You can manually refresh the materialized view or schedule regular refresh tasks.
 
-**refresh_scheme_desc** (optional)
+**refresh_scheme** (optional)
+
+> **NOTE**
+>
+> While creating an asynchronous materialized view, you must specify either the `distribution_desc` or the `refresh_scheme`, or both.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md:214]: you must specify either `distribution_desc` or `refresh_scheme`

File: docs/using_starrocks/Materialized_view.md [hunk 3]
```diff
@@ -148,8 +148,11 @@ GROUP BY order_id;
 
 > **NOTE**
 >
-> - You must specify a bucketing strategy when creating an asynchronous materialized view.
+> - While creating an asynchronous materialized view, you must specify either the `distribution_desc` or the `refresh_scheme`, or both.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/Materialized_view.md:151]: same

File: docs/using_starrocks/Materialized_view.md [hunk 4]
```diff
@@ -148,8 +148,11 @@ GROUP BY order_id;
...
 > - You can set different partitioning and bucketing strategies for an asynchronous materialized view from those of its base tables.
+> - Currently, StarRocks does not support creating asynchronous materialized views with base tables created with the list partitioning strategy.
+> - Currently, asynchronous materialized views do not support the list partitioning strategy.
+> - By default, executing a CREATE MATERIALIZED VIEW statement immediately triggers the refresh task, which can consume certain proportion of the system resources. If you want to defer the refresh task, you can add the REFRESH DEFERRED parameter to your CREATE MATERIALIZED VIEW statement.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/Materialized_view.md:155]: linting error, trailing spaces

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**